fineract-provider/build.gradle

```diff
@@ -21,7 +21,7 @@ Run as:
 gradle clean bootRun'''
 
 group = 'org.apache.fineract'
-buildDir = new File(rootProject.projectDir, "../build")
+buildDir = new File(rootProject.projectDir, "$rootDir/build")
```

Comment:
```suggestion
buildDir = new File(rootProject.projectDir, "build")
```

Reply:
Just tried this change and it seems to fix both Docker and the bootJar/bootWar builds for me...

Reply:
@vidakovic I've locally manually applied @ptuomola proposed 1 line change here (and my #1389; not directly related), and it seems to do the trick - I can `podman build .` it again with that (and had previously locally reproduced that it was broken before with the PR as is, just like on Travis). I hope you therefore don't mind if I Commit Suggestion here - I'm very curious to see if it will also pass on Travis! (I think it will.) If it does, you can still decide if you want to remove `$rootDir/`elsewhere, or just locally make this change (or pull & squash this) and force push it, so that we can merge it. (I won't merge it just yet, even if it passes with this change.)

Reply:
@vorburger Great, let's merge it then if there are no other issues left? Just verified the build again locally; works like a charm. Thanks again @ptuomola for sorting that one out!